src/core/config/loaders/json_loader.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-FileCopyrightText: NVIDIA CORPORATION & AFFILIATES
+ * Copyright (c) 2001-2025 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+ * SPDX-License-Identifier: GPL-2.0-only or BSD-3-Clause
+ *
+ * This software is available to you under a choice of one of two
+ * licenses.  You may choose to be licensed under the terms of the GNU
+ * General Public License (GPL) Version 2, available from the file
+ * COPYING in the main directory of this source tree, or the
```

Comment:
inconsistence `General Public License (GPL) Version 2` and `SPDX-License-Identifier: GPL-2.0-only or BSD-3-Clause`